process/subgroups.md

```diff
@@ -0,0 +1,68 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
+
+- An area of interest that needs a lot of discussion. Creating a subgroup with regular meetings provides more bandwidth for this discussion.
+- An area of interest is only relevant to a small group of participants. In these cases, subgroup meetings give a more focused venue. This way, participants don’t have concerns about wasting the CG’s time with something that the rest of the CG is not engaged in.
+
+To create a subgroup, you should add a poll to a CG meeting agenda with at least 7 days notice. As part of the poll, you should provide information about who will be chairing the subgroup.
+
+## Active Subgroups
+
+| Subgroup  | Chair(s)  | Discussion Repo  | Meetings  |
+|---|---|---|---|
+| GC  | Thomas Lively (@tlively)  | [Repo](https://github.com/WebAssembly/gc)  | [Agendas](https://github.com/WebAssembly/gc/issues?q=is%3Aissue+%22Agenda+for+subgroup+meeting%22)  |
+| SIMD  | Petr Penzin (@penzn), Ng Zhi An (@ngzhian)  | ? | [Agendas](https://github.com/WebAssembly/meetings/tree/main/simd/)  |
+| Stack Switching  | Francis McCabe (@fgmccabe)  | [Repo](https://github.com/WebAssembly/stack-switching)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/stack)  |
+| WASI  | Lin Clark (@linclark), Sam Clegg (@sbc)  | [Repo](https://github.com/WebAssembly/WASI)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/wasi)  |
+
+## Subgroup Leadership Expectations
+
+Subgroups require two kinds of leadership. These roles can be filled by the same person or multiple people:
+
+- Chair(s)—responsible for running the group
+- Champion(s)—responsible for driving technical progress on proposals
```

Comment:
This doc mentions champions here and below, but of course its topic is the role of the chair. It would be nice if we had a similar doc about champions to link to, but unfortunately I don't think we do yet. Maybe the closest thing we have is the [phases](https://github.com/WebAssembly/meetings/blob/main/process/phases.md) doc which mentions the champion in the early stages. Not sure if it's worth adding that here or not.

Reply:
I was expecting that we'd just add a Champions section in the same doc, which was the reason for the invite in the original post for someone to fill that in. If we want to make clear that we intend to fill in documentation later, I could ad a `Champion` section that has a TODO.

Reply:
Ah right, guess I should have read it more carefully 🤦 
Yeah, that sounds good. A TODO is fine, but I'd be fine without it too.

Reply:
We have an in progress effort to document the champion role, once that's published, I think we can add a link to it here. I have a preference to these being documented separately so they are easier to find for new folks wanting to champion proposals.